source/include/third/nrf/generic_gcc_nrf52.ld

```diff
@@ -33,14 +33,21 @@ INCLUDE "nrf_symbols.ld"
  *
  * Update the sections using a recent flash_placement.xml file...
  */
+
+RAM_BLUENET_IPC_START = (RAM_R1_BASE + RAM_APPLICATION_AMOUNT - RAM_BLUENET_IPC_LENGTH);
```

Comment:
Same, quite confusing. 

Actually I see `RAM_APPLICATION_AMOUNT` mainly used to define the end of the ram area which we can use and then subtracting stuff again. 

I think defining something like `RAM_R1_END` to be at the ipc start would simplify a few equations. With, again, just defining a setting that indicates total ram available. Then the developer does not have to calculate this in the config file. Moreover, if we go to different hardware this value is then very easy to change.